locales/usePageLocale.ts

```diff
@@ -12,7 +12,7 @@ const usePageLocale = <NS extends keyof PageLocalesResources>(_ns: NS) => {
   const localeValues = values as Result
 
   function translate<NSK extends keyof Result>(key: NSK): Result[NSK] {
-    return localeValues[key] || (`${key}` as unknown as Result[NSK])
+    return localeValues[key] || (`[${key}]` as unknown as Result[NSK])
```

Comment:
While testing used pl too and noticed wasn't clear that some keys are missing after login in for pl so added this :)